tests/functional/conftest.py

```diff
@@ -37,3 +37,10 @@ def do_submit_known_email_address(email_address):
         set_email_address(email_address)
         browser.find_by_css('form button').click()
     return do_submit_known_email_address
+
+
+@pytest.fixture
+def click_change_email_link_on_dept_confirm(browser):
+    def do_click_change_email_link_on_dept_confirm():
+        browser.find_by_css('#confirm-dept a').click()
```

Comment:
had thought about whether to add an id as it is self contained, but an id would prevent confusion if another link was added in same containing element